common/src/main/java/io/netty/util/internal/ThrowableUtil.java

```diff
@@ -22,6 +22,8 @@
 
 public final class ThrowableUtil {
 
+    private static final Throwable[] EMPTY = new Throwable[0];
```

Comment:
Could move to `EmptyArrays`?

Reply:
I did place it here as it's the only place we need it.

Reply:
I'd still recommend putting it there for consistency's sake.

Reply:
ok ... I am not strong on this one. Let me do it.